src/docfx.website.themes/common/common.js

```diff
@@ -28,16 +28,15 @@ function getHtmlId(input) {
     return input.replace(/\W/g, '_');
 }
 
-function getViewSourceHref(item, gitContribute, gitUrlPattern) {
```

Comment:
or rather, keep the interface unchanged...

---
In reply to: [91878401](https://github.com/dotnet/docfx/pull/979#discussion_r91878401) [](ancestors = 91878401)